coursier.bzl

```diff
@@ -31,12 +34,16 @@ load("@{repository_name}//:jvm_import.bzl", "jvm_import")
 # download them if it's not asked to to resolve "eclipse-plugin".
 _COURSIER_PACKAGING_TYPES = ["jar", "aar", "bundle", "eclipse-plugin"]
 
-# Super hacky :(
-def _strip_packaging_and_classifier(coord):
-    return coord.replace(":jar:", ":").replace(":aar:", ":").replace(":sources:", ":")
+def _strip_packaging_classifier(coord):
+    for packaging_type in _COURSIER_PACKAGING_TYPES + ["pom"]:
```

Comment:
Add "pom" to _COURSIER_PACKAGING_TYPES instead of adding it here

Reply:
_COURSIER_PACKAGING_TYPES is fed into the coursier CLI, which doesn't recognize `pom` as a packaging type. I'll add a comment about it.